client/src/cb-admin/pages/__tests__/Activities.test.js

```diff
@@ -15,79 +15,81 @@ import Activities from '../Activities';
 describe('Activities Component', () => {
   let mock;
 
-  beforeAll(() => {
-    mock = new MockAdapter(axios);
-    mock.onGet('/api/activities/all')
-      .reply(200, { result: [
-        { id: 8,
-          name: 'French Lessons',
-          monday: false,
-          tuesday: true,
-          wednesday: true,
-          thursday: true,
-          friday: true,
-          saturday: false,
-          sunday: false,
-        },
-        { id: 7,
-          name: 'Yoga',
-          monday: false,
-          tuesday: false,
-          wednesday: true,
-          thursday: true,
-          friday: true,
-          saturday: false,
-          sunday: false,
-        },
-        { id: 13,
-          name: 'Skating',
+  if (!process.env.LIVE_API) {
+    beforeAll(() => {
+      mock = new MockAdapter(axios);
+      mock.onGet('/api/activities/all')
+        .reply(200, { result: [
+          { id: 8,
+            name: 'French Lessons',
+            monday: false,
+            tuesday: true,
+            wednesday: true,
+            thursday: true,
+            friday: true,
+            saturday: false,
+            sunday: false,
+          },
+          { id: 7,
+            name: 'Yoga',
+            monday: false,
+            tuesday: false,
+            wednesday: true,
+            thursday: true,
+            friday: true,
+            saturday: false,
+            sunday: false,
+          },
+          { id: 13,
```

Comment:
Is this temporary data based on the resolution of #450 and #453? Either way, I think we should put this data into a separate file(s) so as not to bloat the test specs themselves.